src/mpdclient.nim

```diff
@@ -301,11 +301,17 @@ proc parseReply(s: string): Reply =
     let splits = s.split(':', maxsplit = 1)
     Reply(kind: replyPair, key: splits[0], value: splits[1].strip)
 
-template expectOk(mpd: MPDClient) =
+proc expectOk(mpd: MPDClient) =
   let reply = mpd.readLine.parseReply
   if reply.kind == replyAck:
     raise newException(CatchableError, reply.ack)
 
+proc expectPair(mpd: MPDClient): bool =
```

Comment:
why does this return `bool`?

Reply:
ok, i get it

Reply:
this should be `expectList` and inline it too for good measure

Reply:
In `expectOk` we expect `replyOk`, so to me `expectPair` makes more sense since we want `replyPair`, I'll change it

Reply:
i want to leave the name open for if we need strict expectPair